docs/refs.bib

```diff
@@ -0,0 +1,51 @@
+Mitigations.rst
```

Comment:
I'm in favor of this, but the comments in this file don't add anything from my point of view. In fact, these papers are cited all over the place rather than just in one file. Keeping such a mapping in sync sounds tricky and I don't really see the value it would add since you can just search for the identifier to find where it's cited. Again, just the comments (including paper comments below), everything else is great! I assume you just copied from Google Scholar or somewhere similar?

Reply:
Agreed that we probably don't need comments like this. The entries look like a merge of DBLP versions, which I think is sensible. We shouldn't use Googe Scholar as a source for citations. However, the main decision here is perhaps whether we are okay with the citation ids of the form
```
<last-name-of-first-author><year><first-important-title-word>
```
I'm fine with that.

Reply:
> Keeping such a mapping in sync sounds tricky and I don't really see the value it would add since you can just search for the identifier to find where it's cited. 

Ok that makes sense, I think the step 1 should cover the duplicates, no need for comments. If someone is adding the a new entry, they should check if an entry already exists. I'll modify the PR and the description.

> 1. Check if your required reference already exists in the `docs/refs.bib` file.

Reply:
> I assume you just copied from Google Scholar or somewhere similar?

I used the bibtex entries available through arXiv and modified some of the entries to showcase how different formats look.

> The bibtex entries below look like a merge of DBLP versions, which I think is sensible. We [shouldn't use Googe Scholar](https://scholar.hasfailed.us/) as a source for citations. The main decision is perhaps whether we are okay with the citation ids of the form

Thank you for the reference, I did not know that about Google Scholar. I used the ones available through arXiv, they link to the DBLP hosted records. For example - [arXiv Page Fairlearn](https://arxiv.org/abs/1803.02453) and its [DBLP Bibtex](https://dblp.uni-trier.de/rec/journals/corr/abs-1803-02453.html?view=bibtex). However, they always link back to the arXiv page for the paper.

**Takeaway**: It might be helpful to establish where we source our bibtex snippets from.